<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleDialog</class>
 <widget class="QDialog" name="ModuleDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modules</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="m_treeWidget">
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Instance Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Module</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="m_upButton">
       <property name="text">
        <string>Move Up</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/arrow_up.png</normaloff>:/images/arrow_up.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_downButton">
       <property name="text">
        <string>Move Down</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/arrow_down.png</normaloff>:/images/arrow_down.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_addButton">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_deleteButton">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/time_go.png</normaloff>:/images/time_go.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="m_buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModuleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>657</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModuleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>662</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_treeWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>ModuleDialog</receiver>
   <slot>selectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_upButton</sender>
   <signal>clicked()</signal>
   <receiver>ModuleDialog</receiver>
   <slot>moveUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_downButton</sender>
   <signal>clicked()</signal>
   <receiver>ModuleDialog</receiver>
   <slot>moveDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addButton</sender>
   <signal>clicked()</signal>
   <receiver>ModuleDialog</receiver>
   <slot>addModule()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>ModuleDialog</receiver>
   <slot>removeModule()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_resetButton</sender>
   <signal>clicked()</signal>
   <receiver>ModuleDialog</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_treeWidget</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>ModuleDialog</receiver>
   <slot>editItem(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectionChanged()</slot>
  <slot>moveUp()</slot>
  <slot>moveDown()</slot>
  <slot>addModule()</slot>
  <slot>removeModule()</slot>
  <slot>reset()</slot>
  <slot>editItem(QTreeWidgetItem*,int)</slot>
 </slots>
</ui>
